src/main/java/edu/harvard/iq/dataverse/dataaccess/InputStreamIO.java

```diff
@@ -138,4 +139,12 @@ public OutputStream getOutputStream() throws IOException {
         throw new UnsupportedDataAccessOperationException("InputStreamIO: there is no output stream associated with this object.");
     }
 
+   
+
+    @Override
+    public InputStream getAuxFileAsInputStream(String auxItemTag) {
```

Comment:
Ok but it throws `UnsupportedOperationException` so I guess it isn't needed. It seems confusing to me but not confusing enough to prevent this from going to QA.